lib/country_select/tag_helper.rb

```diff
@@ -13,9 +13,8 @@ def country_option_tags
         option_tags = options_for_select(priority_countries_options, option_tags_options)
         option_tags += html_safe_newline + options_for_select([priority_countries_divider], disabled: priority_countries_divider)
 
-        if priority_countries_options.map(&:second).include?(option_tags_options[:selected])
-          option_tags_options[:selected] = nil
-        end
+        option_tags_options[:selected] = [option_tags_options[:selected]] unless option_tags_options[:selected].kind_of?(Array)
```

Comment:
IMHO it is understandable enough right now, but it's totally up to you, so let me know if you prefer a different way of doing it. 
Single select being cast as an array still works because the idea is treat it always as a "multiple selection", but only one is selected. I mean, it's not a coincidence that it works fine for a single select, isn't it?